### ISSUE TITLE
Estuary Implementation talk

### ISSUE THREAD

**Issue by b5** [ipfs-shipyard/ipfs-thing-2022#132]
Three members of the Estuary team are here. @jlogelin can you confirm you're able to give a 5-10 minute talk about Estuary as an IPFS implementation? If so we'll need to get you re-added to the schedule

**Comment by b5** [comment]
cc @jbenet

**Comment by jbenet** [comment]
@jlogelin great! added back in #133

**Comment by jlogelin** [comment]
Yep - I'll be there. :) Corinne Bernette would like to zoom in and do some intro slides as well if possible.
